src/db.rs

```diff
@@ -1368,6 +1936,16 @@ pub enum Operation<Key, Value> {
 	/// Increment the reference count counter of an existing value for a given key.
 	/// If no value exists for the key, this operation is skipped.
 	Reference(Key),
+
+	/// Insert a new tree into a MultiTree column using root key and node structure.
+	InsertTree(Key, NewNode),
+
+	/// Increment the reference count of a tree (at root Key) from a MultiTree column.
+	ReferenceTree(Key),
```

Comment:
Is it the same as Reference(Key)?

Reply:
Let's keep them separate for a more clear distinction in the API